### PR TITLE
Onboard Management.Network to version 09-01-2017

### PR DESCRIPTION
**Pull request by murilogr**
# Description

This change was autogenerated from the Swagger changes for the new version of the Network API:

[https://github.com/Azure/azure-rest-api-specs/pull/1661/commits](https://github.com/Azure/azure-rest-api-specs/pull/1661/commits)

These changes have been approved by ARM and REST teams and are merged into "current" branch.

The changes include:
- Application Security Group
- Secure VNETs
- ILB HA
- Small API fixes

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by murilogr** [comment]
Please provide guidelines for updating the version.

**Review by shahabhijeet — COMMENTED** [review]
Update version so that you can publish the nuget.
Add < PackageReleaseNotes > in .csproj
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Microsoft.CognitiveServices.Language/Microsoft.CognitiveServices.Language.csproj#L11

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Update version so that you can publish the nuget.
Add < PackageReleaseNotes > in .csproj
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Microsoft.CognitiveServices.Language/Microsoft.CognitiveServices.Language.csproj#L11

**Comment by murilogr** [comment]
@shahabhijeet I updated the PR with the requested changes.

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -4,8 +4,10 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-    <Version>14.1.0-preview</Version>
-    <PackageTags>Microsoft Azure Network management;Network;Network management;windowsazureofficial</PackageTags>    
+    <Version>14.2.0-preview</Version>
+    <PackageTags>Microsoft Azure Network management;Network;Network management;windowsazureofficial</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the Network SDKs. It includes the support for application security groups (ASGs), secure VNETs, ILB HA,
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Network SDK is one of the top 5 RPs. Providing details that helps to understand what the new version has is very helpful for developers who are trying to develop using Netwok SDK.
What does ILB HA mean?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@murilogr Please address the feedback or this PR will be closed by eow

**Comment by murilogr** [comment]
@shahabhijeet These changes were done in the private repo. I'm deactivating this PR, unless I see there's any diff later.

**Pull request closed without merging**